include/dem/distributions.h

```diff
@@ -48,20 +48,25 @@ class NormalDistribution : public Distribution
    * @brief The constructor stores the parameters necessary to define the normal
    * distribution.
    *
-   * @param d_average Average diameters for a certain normal distribution.
-   * @param d_standard_deviation Standard deviation of the diameter for a certain
-   * normal distribution.
-   * @param prn_seed Pseudo-random number seed for the diameter generation.
+   * @param[in] d_average Average diameters for a certain normal distribution.
+   * @param[in] d_standard_deviation Standard deviation of the diameter for a
+   * certain normal distribution.
+   * @param[in] prn_seed Pseudo-random number seed for the diameter generation.
+   * @param[in] min_cutoff Minimum cutoff diameter.
+   * @param[in] max_cutoff Maximum cutoff diameter.
    */
   NormalDistribution(const double       &d_average,
                      const double       &d_standard_deviation,
-                     const unsigned int &prn_seed);
+                     const unsigned int &prn_seed,
+                     const double       &min_cutoff,
+                     const double       &max_cutoff);
 
   /**
    * @brief Carries out the size sampling of each particle inserted at an insertion
    * time step for the normal distribution.
    *
-   * @param particle_number Number of particle inserted at a given insertion time step.
+   * @param[in] particle_number Number of particle inserted at a given insertion
+   * time step.
    */
   void
   particle_size_sampling(const unsigned int &particle_number) override;
```

Comment:
I would be tempted to rename this parameter number_of_particles but it's maybe not relevant here.